### ISSUE TITLE
Bad Icon

### ISSUE THREAD

**Issue by GoogleCodeExporter** [StonedRightMeow17/itransmission#5]
```
Please switch to the previous one, this one sucks!
```

Original issue reported on code.google.com by `xlife018` on 2 Jun 2011 at 9:15

**Comment by GoogleCodeExporter** [comment]
```
my friend how did u install the 0.4 ipa file? thank you..
```

Original comment by `panepist...@gmail.com` on 2 Jun 2011 at 9:56

**Comment by GoogleCodeExporter** [comment]
```
I installed it from the insanelyi repo not from here!
```

Original comment by `xlife018` on 4 Jun 2011 at 1:46

**Comment by GoogleCodeExporter** [comment]
```
yes but in the repo it's 0.3 not 0.5 :)
```

Original comment by `panepist...@gmail.com` on 6 Jun 2011 at 11:07

**Comment by GoogleCodeExporter** [comment]
```
No its 0.4, 0.5 will be up as well as soon as sull (repo admin) sees this update
```

Original comment by `xlife018` on 6 Jun 2011 at 6:51

**Comment by GoogleCodeExporter** [comment]
```
One can get around this by editing "info.plist" in 
"/Applications/iTransmission" change. 
Once the file is open, change "UIApplicationExitsOnSuspend" to true. This will 
suspend it when in the background. Once this is done you can use Backgrounder 
to have your torrents run in the background. 
```

Original comment by `3radicat...@gmail.com` on 13 Jun 2011 at 3:22

**Comment by GoogleCodeExporter** [comment]
```
Seriously? If you have a better icon that I can use freely, please let me know. 
I'm sure I'll be happy to install new icons. 
```

Original comment by `ccp0...@gmail.com` on 19 Jun 2011 at 1:30
- Changed state: **WontFix**

**Comment by GoogleCodeExporter** [comment]
```
Ver 4 has the ugriest icon ever. It biger than the rest of IOS icons, like 
there is no alpha mask.
```

Original comment by `lwkris...@gmail.com` on 28 Apr 2014 at 12:58

Attachments:
- [IMG_0392.PNG](https://storage.googleapis.com/google-code-attachments/itransmission/issue-5/comment-7/IMG_0392.PNG)
